coriolis/tasks/osmorphing_tasks.py

```diff
@@ -25,11 +25,22 @@ def run(self, ctxt, instance, origin, destination, task_info,
             task_info['osmorphing_connection_info'])
         osmorphing_info = task_info.get('osmorphing_info', {})
 
+        user_scripts = task_info.get("user_scripts")
+        instance_script = None
+        if user_scripts:
+            instance_script = user_scripts.get("instances", {}).get(instance)
+            if not instance_script:
+                os_type = osmorphing_info.get("os_type")
+                if os_type:
+                    instance_script = user_scripts.get(
```

Comment:
I'm not entirely sure how we've implemented this in the UI but we should be sure that it's clear to the user that any per-instance scripts will completely override the global ones (I was under the initial assumption that the globals would always get executed, so it's something I imagine some users would make thus supposition too)